File: official/docs/node/shipments/create_shipment_with_options.js
const Easypost = require('@easypost/api');
const api = new Easypost('EASYPOST_API_KEY');

const shipment = new api.Shipment({
  parcel: {
    id: 'prcl_...',
  },
  to_address: {
    id: 'adr_...',
  },
  from_address: {
    id: 'adr_...',
  },
  options: {
    print_custom_1: 'Custom label message',
  },
});

shipment.save().then(console.log);

File: official/docs/go/shipments/create_shipment_with_options.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	toAddress, err := client.GetAddress("shp_...")
	fromAddress, err := client.GetAddress("adr_...")
	parcel, err := client.GetParcel("prcl_...")

	shipment, err := client.CreateShipment(
		&easypost.Shipment{
			ToAddress:   toAddress,
			FromAddress: fromAddress,
			Parcel:      parcel,
			Options: &easypost.ShipmentOptions{
				AddressValidationLevel: "0",
			},
		},
	)

	fmt.Println(shipment)
}
